proxy-libs/helioslogrus/go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/helios/go-sdk/proxy-libs/helioslogrus
+
+go 1.18
+
+require (
+	github.com/sirupsen/logrus v1.9.0
+	go.opentelemetry.io/otel v1.13.0
```

Comment:
We had problems with v1.12.0, right? Don't remember where but it broke one of our integrations